src/sensors/src/Accelerometer.cpp

```diff
@@ -139,6 +138,20 @@ SensorType Accelerometer::getSensorType() const
     return ACCELEROMETER;
 }
 
+bool Accelerometer::getLinkSensorTransform(Transform& link_H_sensor) const
+{
+    if( this->pimpl->parent_link_index < 0)
```

Comment:
The method had the signature with the `bool` because it was used in the FT sensors, where the users was passing to the function the link index (one of the two that are attached to the FT sensor) with respect to which he wanted the transform. In this case the method is just returning an attribute of the class, so I think that most users don't expect all the getters of a class to stop working if an attribute as an invalid value.